eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ConfigResolver/SiteAccessGroupConfigResolver.php

```diff
@@ -33,6 +86,16 @@ protected function resolverHasParameter(SiteAccess $siteAccess, string $paramNam
         return false;
     }
 
+    protected function resolverHasParameterForGroup(SiteAccessGroup $siteAccessGroup, string $paramName, string $namespace): bool
+    {
+        $groupScopeParamName = $this->resolveScopeRelativeParamName($paramName, $namespace, $siteAccessGroup->getName());
+        if ($this->container->hasParameter($groupScopeParamName)) {
+            return true;
+        }
+
+        return false;
```

Comment:
Would be better to use `ParameterBag` instead of a whole container. Ref. https://symfony.com/blog/new-in-symfony-4-1-getting-container-parameters-as-a-service

Also:

```suggestion
return $this->container->hasParameter($groupScopeParamName);
```